README.rst

```diff
@@ -60,7 +60,7 @@ Synopsis
 
   usage: autospec.py [-h] [-g] [-n NAME] [-v VERSION]
                      [-a [ARCHIVES [ARCHIVES ...]]] [-l] [-b] [-c CONFIG]
-                     [-t TARGET] [-i] [-p] [--non_interactive] [-C]
+                     [-t TARGET] [-i] [-p] [--non_interactive] [-C] [--infile]
```

Comment:
should be `[--infile INFILE]` to show it needs an argument.

Reply:
Do you want to keep [url] as its own line? Is that intentional for formatting?

Reply:
it's fine where it is.